### ISSUE TITLE
Publish command with -o/--output ignores publishOptions include/exclude

### ISSUE THREAD

**Issue by rinobatin** [dotnet/cli#3739]
**Removing `-o` works fine.**

```
//project.json
...
"publishOptions": {
    "exclude": [
      "Areas/**/Controllers",
      "Areas/**/*.cs"

    ],
    "include": [
      "wwwroot",
      "Areas",
      "Views",
      "appsettings.json",
      "web.config"
    ]
  },

  "scripts": {
    "precompile": [ ],
    "postcompile": [ ],
    "prepublish": [ ],
    "postpublish": [ "dotnet publish-iis --publish-folder %publish:OutputPath% --framework %publish:FullTargetFramework%" ]
  },...
```
## Steps to reproduce
1. Run `dotnet -v publish -o .\.deploy\Dev\ --configuration Release --version-suffix beta.1.0.0`
## Expected  behavior

Include files and folder should be in .\deploy\Dev
## Actual behavior

Include files and folder are missing.
## Environment data

`dotnet --info` output:

```
.NET Command Line Tools (1.0.0-preview2-003121)

Product Information:
 Version:            1.0.0-preview2-003121
 Commit SHA-1 hash:  1e9d529bc5

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.3.9600
 OS Platform: Windows
 RID:         win81-x64
```

**Comment by jchannon** [comment]
Can also confirm I see this issue

**Comment by jchannon** [comment]
Similar to this https://github.com/dotnet/cli/issues/3745

**Comment by moander** [comment]
Absolute paths did *not* work for me using `1.0.0-preview2-003121` on OS X.

**Comment by piotrpMSFT** [comment]
I just closed #3745 after testing it out on preview3 bits. The whole publish infrastructure was re-written for csproj and is doing much cleaner path handling. Since this seems to be a dupe, I'm closing the issue. Try it out with Preview3 and let us know if you see any additional problems!
